pkg/sensors/sensors.go

```diff
@@ -241,3 +257,18 @@ func progsCleanup() {
 func AllPrograms() []*program.Program {
 	return append([]*program.Program{}, allPrograms...)
 }
+
+// sortSensors sort the sensors to enforce orderging constrains
+func sortSensors(sensors []SensorIface) {
+	sort.Slice(sensors, func(i, j int) bool {
```

Comment:
seems ok, but I was wondering we want something more generic, like adding sort priority for the sensor.. could be follow up

Reply:
Agreed. I think eventually we would want something more generic. I'm not sure what it would be though. Having a number seems hard to do.